packages/datatrak-web/src/views/AccountSettingsPage/RequestCountryAccessSection/RequestCountryAccessSection.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { AccessGrantedCountryList } from './AccessGrantedCountryList.tsx';
```

Comment:
Just looking through this - is there a reason for adding the file extension here btw?

Reply:
Nope! Fixed! Thanks!